src/compiler/crystal/syntax/ast.cr

```diff
@@ -228,6 +228,24 @@ module Crystal
 
     def_equals value.to_f64, kind
     def_hash value, kind
+
+    def self.kind_from_number(number : Number)
```

Comment:
Is this the only usage of `kind_from_number`? or there are plans to reuse it somewhere else?

Asking this because either make it protected/private or add `:nodoc:`.

Thank you.

Reply:
Atm it is the only usage, I don't know if it there is a different use case.

Compiler types and methods are not really documented anyway, so there is not much gain from adding `:nodoc:`.

I don't think there are any particular guidelines about this for the compiler.